src/main/xerus/monstercat/api/Covers.kt

```diff
@@ -5,24 +5,29 @@ import org.apache.http.HttpEntity
 import org.apache.http.client.methods.HttpGet
 import xerus.ktutil.replaceIllegalFileChars
 import xerus.monstercat.cacheDir
+import java.io.File
 import java.io.InputStream
 import java.net.URI
 
 object Covers {
 	
 	private val coverCacheDir = cacheDir.resolve("cover-images").apply { mkdirs() }
+	private val largeCoverCacheDir = cacheDir.resolve("large-cover-images").apply { mkdirs() }
```

Comment:
please rename the dir variables to match the function names, i.e. thumbnail and cover

Reply:
Renamed in 5a080655c024620044533366bd60b61dc00493df